frontend/test/metabase/scenarios/visualizations/drillthroughs/dash_drill.cy.spec.js

```diff
@@ -1,122 +1,87 @@
 // Imported from drillthroughs.e2e.spec.js
 import { restore, signInAsAdmin, withSampleDataset } from "__support__/cypress";
 
-let dash_name_var;
-let dash_num;
-
-function addCardToNewDash(dash_name, card_id) {
-  dash_name_var = dash_name;
-  cy.request("POST", "/api/dashboard", {
-    name: dash_name,
-    parameters: [],
-  });
-  cy.request("POST", `/api/dashboard/${dash_num}/cards`, {
-    id: dash_num,
-    cardId: card_id,
-    parameter_mappings: [],
-  });
-}
+// This question is part of our pre-defined data set used for testing
+const Q2 = {
+  name: "Orders, Count",
+  id: 2,
+};
 
 describe("scenarios > visualizations > drillthroughs > dash_drill", () => {
-  describe("title click action", () => {
-    before(restore);
-
+  describe("card title click action", () => {
     describe("from a scalar card", () => {
-      before(() => {
-        cy.server();
-        cy.route("POST", "/api/card/*/query").as("card");
+      const DASHBOARD_NAME = "Scalar Dash";
 
+      before(() => {
+        restore();
         signInAsAdmin();
 
-        // Make the second question scalar
-        cy.visit("/question/1");
```

Comment:
- This test was actually updating question `#1`, instead of `#2` like the comment suggested.
- It is later adding (unchanged) question `#2` to a newly created dashboard (note that question `#2` is not even scalar)
- The final assertion `cy.contains("18,760");` worked simply because Cypress was able to find that string regardless of tabular vs scalar presentation.